src/big_requests/request.rs

```diff
@@ -0,0 +1,15 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+extern crate self as xrb;
+
+use crate::{big_requests::reply, message::Request};
+use derivative::Derivative;
+use xrbk_macro::derive_xrb;
+
+derive_xrb! {
+	#[derive(Derivative, Debug, X11Size, Readable, Writable)]
+	#[derivative(Hash, PartialEq, Eq)]
+	pub struct BigRequestsEnable: Request(0) -> reply::BigRequestsEnable {}
```

Comment:
I assume the `0` is a stand-in for the dynamic major opcode?  Or is it the minor opcode (I would assume the extension has no minor opcodes because it only has one request?)?  If it is a stand-in for the dynamic major opcode, maybe replace it with `todo!()` if that works with the macro?
```suggestion
	pub struct BigRequestsEnable: Request(todo!("extensions use dynamic major opcodes")) -> reply::BigRequestsEnable {}
```

Reply:
it was indeed a placeholder